mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/MapInstance.java

```diff
@@ -386,7 +386,7 @@ public void onDestroy() {
 
```

Comment:
Relax, it is still work in progress.

I created a pull request, because the number of changes is huge and I want people to see where this is going.

It is not ready to be merged.